.github/workflows/go_tests.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Delete old coverage
         run: "cd sdks/go/pkg && rm -rf .coverage || :"
       - name: Run coverage
-        run: cd sdks/go/pkg && go test -coverprofile=coverage.txt -covermode=atomic ./...
+        run: cd sdks && go test -coverprofile=coverage.txt -covermode=atomic ./go/pkg/... ./go/container/... ./java/container/... ./python/container/... ./typescript/container/...
```

Comment:
Question: does codecov count uncovered non-go files (e.g. https://github.com/apache/beam/blob/master/sdks/java/container/license_scripts/pull_licenses_java.py)? I'm wondering if it makes sense to run container tests separately outside of the context of codecov (I'm leaning towards thinking that they should be included so this is probably a no-op).

Reply:
It will ignore those files, because it's unaware of files that don't receive any coverage numbers/changes.

I think we also don't have the covbot enabled for these directories either. But I'm less concerned about that than I am about the unit tests being run at all.

Reply:
Enabling covbot here would be out of scope for this change I think. The go tests is different because it's otherwise hard to ensure an action will run, without having a change that ensures the action runs...

Reply:
SGTM - was meant to be nonblocking, but I realize now I didn't specify that